Backend/src/ml_lib/sub_models/qaqc_main_model.py

```diff
@@ -0,0 +1,92 @@
+import pandas as pd
+import csv
+import re
+import os
+
+from sentence_transformers import SentenceTransformer
+from sklearn.decomposition import PCA
+from sklearn.manifold import TSNE
+
+from ..shared import IsbitClassifierModel
+
+
+class QaqcMainModel(IsbitClassifierModel):
+
+    def _format_data(self, file_name: str) -> None:
```

Comment:
I saw that the `_format_data` looks identical to the test model version, do we know if  it will be used for a lot of different models/all the models?

If so, could we break out the functionality into a separate file? Making less redundant code?

Reply:
Sounds good